### ISSUE TITLE
Clarify documentation around location/availability of source code for vscode-remote extensions

### ISSUE THREAD

**Issue by kdvolder** [microsoft/vscode-remote-release#1402]
I am having a hard time finding the source code for the various vscode-remote extensions. (E.g. the docker and ssh remote support extensions).

I was initially under the impression/assumption that they are open-sourced (after all I am filing this very ticket on github which is a platform for open-source projects). But I am beginning to suspect that perhaps they are actually closed-source. Either way the docs are not clear about this and could be improved to provide clarity around where/if source code is available.

So... If the extensions are closed-source, please consider some of the following:

- making them open source (not sure this is possible but we can always ask/suggest :-)
- adding a note to the [FAQ](https://code.visualstudio.com/docs/remote/faq) and/or github [readme](https://github.com/microsoft/vscode-remote-release/blob/master/README.md) to make it clear the extensions are closed-source.

If the extensions *are* actually open-source and I just have not been looking in the right places for the source code, then maybe consider adding some pointers in the FAQ and README to make the source code easier to find.

PS: This isn't strictly a feature request nor a Bug. More of request for improving the docs. I don't see a category for that specifically so I choose "Feature request" as that feels like the closest match.

**Comment by Chuxel** [comment]
Sorry to hear you missed that detail! While many marketplace extensions are OSS, some both Microsoft and non-Microsoft extensions provide proprietary or paid features and therefore use different licenses including the Remote Development extensions. GitHub is also a central location to communicate with developers like yourself, so many proprietary products include feedback repositories like this one.  Sorry if that cause confusion!

We value transparency, so we have included this information in several spots to try to cover our bases:

1. FAQ: https://code.visualstudio.com/docs/remote/faq#_license-and-privacy
2. A summary of licenses for MS published extensions can be found here: https://code.visualstudio.com/docs/supporting/oss-extensions
3. The marketplace links to the license as does this repository.

Hopefully that helps!

**Comment by kdvolder** [comment]
> Hopefully that helps!

Thanks yes it does. I did look in the FAQ but somehow totally missed that section about license. The section is clear and detailed and I think it covers my questions. So I think its okay and just close this issue. If however you do feel like making this somehow even more clear or easier to find. Here are two ideas:

- Add a section, a link or a license file in the github repo or readme (github repo would be the first place I would check when I am looking for source code).
- Ensure the license section shows up on the page outline of the faq page. i.e. when we open the FAQ page currently, there is a navigation outline on the right margin of the page, but the section about 'licence and privacy' does not show up in that outline. It would have been easier to find if it did show up in there. 

Anyhow, these are just suggestions for you to take or leave at your discretion. What you have there is already clear and I only have myself to blame for not checking the FAQ more carefully.

**Comment by Chuxel** [comment]
Thanks so much for the feedback!  Definable don't want anyone being surprised.

> Ensure the license section shows up on the page outline of the faq page. i.e. when we open the FAQ page currently, there is a navigation outline on the right margin of the page, but the section about 'licence and privacy' does not show up in that outline. It would have been easier to find if it did show up in there.

Interesting - we are A/B testing some site theme updates that might have caused a problem.  Was the theme you saw dark?  Here's what should appear in the right nav:

![image](https://user-images.githubusercontent.com/312252/64741176-7beb6a80-d4ac-11e9-97c7-224ca5ae78c0.png)

**Comment by kdvolder** [comment]
Actually yes this outline is there now. I'm not sure it was there before. But It probably was and what I said before '  i.e. when we open the FAQ page currently, there is a navigation outline on the right margin of the page' this was a bit of typo on my behalf. I never actually seen the outline on the right side because I was really only looking at the outline on the left side. Yes I did write 'right margin' but it's typo, I typed 'right' but meant to say 'left').

So... I think your outline was there all along but I wasn't looking at it. I think our brains (or mine at least) have been trained to ignore whole sections of pages (by various social media and advertising content) and I think that my subconcious pattern matcher had previously blocked out the entire right margin as 'social media cruft' because there are some twitter icons and such in that section.

I seem to be estabiishing a pattern of repeatedly asking for improvements that are already there, which is a bit embarassing :-) If my confusing provides you with any inspiration to make things better, great. But feel free to close this train-wreck of a ticket without further action :-)
